runtime/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/config/MqttKafkaBindingConfig.java

```diff
@@ -64,4 +102,58 @@ public String16FW retainedTopic()
     {
         return options.topics.retained;
     }
+
+    public List<MqttKafkaRouteConfig> bootstrapRoutes()
+    {
+        final List<MqttKafkaRouteConfig> bootstrapRoutes = new ArrayList<>();
+        routes.forEach(r ->
+        {
+            if (options.clients.stream().anyMatch(r::matchesClient))
+            {
+                bootstrapRoutes.add(r);
+            }
+        });
+        return bootstrapRoutes;
+    }
+
+    private List<Function<String, String>> asAccessor(
+        List<String> clients)
+    {
+        List<Function<String, String>> accessors = new ArrayList<>();
+
+        if (clients != null)
+        {
+            for (String client : clients)
+            {
+                Matcher topicMatch =
+                    Pattern.compile(client.replace("{identity}", "(?<identity>[^\\s/]+)").replace("#", ".*"))
+                        .matcher("");
+
+                Function<String, String> accessor = DEFAULT_CLIENTS;
+                accessor = orElseIfNull(accessor, topic ->
```

Comment:
Why not just assign `accessor` to the lambda starting with `topic -> ` directly?